guides/common/modules/proc_adding-kubevirt-connection.adoc

```diff
@@ -12,28 +12,30 @@ Use this procedure to add {KubeVirt} as a compute resource in {Project}.
 # {foreman-installer} --enable-foreman-plugin-kubevirt
 ----
 
-. Generate a bearer token to use for HTTP and HTTPs authentication.
-On the {KubeVirt} server, list the secrets that contain tokens:
+. Obtain a token to use for HTTP and HTTPs authentication:
+
+.. Log in to the {Kubernetes} cluster and list the secrets that contain tokens:
 +
+[options="nowrap" subs="+quotes,attributes"]
 ----
-# kubectl get secrets
+$ {kubevirt-command} get secrets
 ----
 
-. List the token for your secret:
+.. Obtain the token for your secret:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# kubectl get secrets _YOUR_SECRET_ -o jsonpath='{.data.token}' | base64 -d | xargs
+$ {kubevirt-command} get secrets _YOUR_SECRET_ -o jsonpath='{.data.token}' | base64 -d | xargs
 ----
-+
-Make a note of this token to use later in this procedure.
+
+.. Record the token to use later in this procedure.
 
 . In the {ProjectWebUI}, navigate to *Infrastructure* > *Compute Resources*, and click *Create Compute Resource*.
 . In the *Name* field, enter a name for the new compute resource.
 . From the *Provider* list, select *{KubeVirt}*.
 . In the *Description* field, enter a description for the compute resource.
-. In the *Hostname* field, enter the FQDN, hostname, or IP address of the {KubeVirt} server that you want to use.
+. In the *Hostname* field, enter the FQDN, hostname, or IP address of the {Kubernetes} cluster.
 . In the *API Port* field, enter the port number that you want to use for provisioning requests from {Project} to {KubeVirt}.
-. In the *Namespace* field, enter the user name of the {KubeVirt} virtual cluster that you want to use.
+. In the *Namespace* field, enter the user name of the {Kubernetes} cluster.
```

Comment:
This needs to be checked. A Kubernetes namespace is not the same as a user name.

Reply:
@ares Do you know what the **Namespace** field is for? It seems strange for a user to fill in a namespace when the secret is a cluster-scoped resource.

Reply:
TBD